contentcuration/contentcuration/api.py

```diff
@@ -5,20 +5,22 @@
 import json
 import logging
 import os
-import shutil
 from cStringIO import StringIO
 
-import contentcuration.models as models
 from django.conf import settings
 from django.core.exceptions import SuspiciousOperation
 from django.core.files.storage import default_storage
-from django.db.models import Count, Q, Sum
```

Comment:
Hmm, is this not allowed anymore?  :crying_cat_face:

Reply:
might encourage this style:

```python
from django.db import models
models.Sum(...)
```